h2o-py/tests/testdir_munging/pyunit_distance.py

```diff
@@ -0,0 +1,50 @@
+from builtins import range
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+
+import numpy as np
+
+def distance_test():
+
+    iris_h2o = h2o.import_file(path=pyunit_utils.locate("smalldata/iris/iris.csv"))
+    iris_np = np.genfromtxt(pyunit_utils.locate("smalldata/iris/iris.csv"),
+                            delimiter=',',
+                            skip_header=1,
+                            usecols=(0, 1, 2, 3))
+
+    references = iris_h2o[10:150,0:4]
+    queries    = iris_h2o[0:10,0:4]
+
+    D = references.distance(queries, "l1")
+    assert(D.min() >= 0)
+    D = references.distance(queries, "l2")
+    assert(D.min() >= 0)
+    D = references.distance(queries, "cosine")
+    assert(D.min() >= -1)
+    assert(D.max() <= 1)
+    D = references.distance(queries, "cosine_sq")
+    assert(D.min() >= 0)
+    assert(D.max() <= 1)
+
+    A = references.distance(queries, "l2")
+    print(A)
+
+    B = references.distance(queries, "cosine")
+    print(B)
+
+    a = queries.distance(references, "l2")
+    print(a)
+
+    b = queries.distance(references, "cosine")
+    print(b)
+
```

Comment:
A bunch of print statements above without context. Better for debugging if you add a print of what is being done, i.e., Calculating l2 distance metric then print(a)